test/11_array.jl

```diff
@@ -531,9 +534,14 @@ for isordered in (false, true)
             cca2 = compact(ca2)
             r = vcat(cca1, cca2)
             @test r == vcat(a1, a2)
+            @test isa(cca1, CategoricalArray{Int,1,UInt8})
+            @test isa(cca2, CategoricalArray{Int,1,UInt8})
             @test isa(r, CategoricalArray{Int,1,CategoricalArrays.DefaultRefType})
             @test isa(vcat(cca1, ca2), CategoricalArray{Int,1,CategoricalArrays.DefaultRefType})
+            @test ordered(r) == false
+            @test sort(levels(r)) == collect(3:300)
```

Comment:
But I'm not testing the order of levels. The order of levels doesn't matter if ordered = false